infrastructure/variables.tf

```diff
@@ -34,3 +34,33 @@ variable "jenkins_AAD_objectId" {
 }
 
 variable "aks_subscription_id" {}
+
+variable "action_group_name" {
+  description = "The name of the Action Group to create."
+  type        = string
+  default     = "wa-support"
+}
+
+variable "email_address_key" {
+  description = "Email address key in azure Key Vault."
+  type        = string
+  default     = "db-alert-monitoring-email-address"
+}
+
+variable "cpu_threshold" {
+  default     = 5
+  type        = number
+  description = "Average CPU utilisation threshold"
+}
+
+variable "memory_threshold" {
+  default     = 5
```

Comment:
Can I assume these values are currently just test thresholds?

Reply:
yep, just to trigger alerts easily, will be changed to a qa testers email before going live, defaults are 80 set in charts.